src/config/reader.rs

```diff
@@ -41,6 +41,10 @@ impl ConfigReader {
     async fn read_file<T: ToString>(&self, file: T) -> anyhow::Result<FileRead> {
         // Is an HTTP URL
         let content = if let Ok(url) = Url::parse(&file.to_string()) {
+            if cfg!(windows) && !url.scheme().starts_with("http") {
+                let content = self.runtime.file.read(&file.to_string()).await?;
+                return Ok(FileRead { content, path: file.to_string() });
+            }
```

Comment:
why do we need this?

Reply:
Reqwest doesn't support reading local file paths on windows. 

https://github.com/seanmonstar/reqwest/issues/178

Reply:
I don't think we support `file://` anyways.. this is redundant I think